benefits/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,47 @@
+{% extends "admin/login.html" %}
+
+{% block title %}
+  {% if form.new_password1.errors or form.new_password2.errors %}Error:{% endif %}
+  {{ block.super }}
+{% endblock title %}
+
+{% block content %}
+  {% if validlink %}
+    <h1>Reset your password</h1>
+
+    {% if form.non_field_errors %}
+      {% for error in form.non_field_errors %}<p class="errornote">{{ error }}</p>{% endfor %}
+    {% endif %}
+
+    <form method="post">
+      {% csrf_token %}
+      <input class="hidden" autocomplete="username" value="{{ form.user.get_username }}">
+      <div class="form-row field-password1">
+        {{ form.new_password1.errors }}
+        <div class="flex-container">
+          <label for="id_new_password1">Enter new password</label>
+          {{ form.new_password1 }}
+        </div>
+      </div>
+      <div class="form-row field-password2">
+        {{ form.new_password2.errors }}
+        <div class="flex-container">
+          <label for="id_new_password2">Confirm new password</label>
+          {{ form.new_password2 }}
+        </div>
+      </div>
+      <div class="submit-row">
+        <input type="submit" value="Reset password">
+      </div>
+
+      {% if request.recaptcha %}
+        {% include "core/includes/recaptcha.html" %}
+      {% endif %}
+    </form>
+  {% else %}
+    <p>This password link is invalid, possibly because it's expired or has already been used.</p>
+    <p class="password-reset-link">
```

Comment:
Is this an existing Django class? I didn't see it on the previous iteration of this section (before the links) and it isn't in our stylesheet. I'm actually not seeing any styles applied to this class -- do we need it?